frontend/pages/redeem/[urlToken]/index.tsx

```diff
@@ -136,6 +143,15 @@ const Redeem: NextPage = () => {
       </Head>
 
       <Container component="main" maxWidth="md">
+        {isCouponNotRedeemed && (
+          <IconButtonWithTooltip
+            sx={isMobile ? mobileHelpButtonSx : desktopHelpButtonSx}
+            icon={<HelpOutlineIcon />}
+            tooltip="Instructions"
+            onClick={() => setOpenInstructions(true)}
+          />
+        )}
```

Comment:
The button does not work for me for mobile view.


https://user-images.githubusercontent.com/53614454/197392299-66637369-7f13-4021-9d2f-320a9e4079d7.mp4

Reply:
![image](https://user-images.githubusercontent.com/53614454/197392359-256404f4-756d-4180-ba97-e269e86ba396.png)

The reason is the stepper is covering the button and the stepper has zIndex: 1.

Easy fix by setting zIndex of button to be > 1